ctypesgen/__main__.py

```diff
@@ -38,18 +38,16 @@ def main(givenargs=None):
     )
 
     # Parameters
+    parser.add_argument("headers", nargs="+", help="Sequence of header files")
     parser.add_argument(
-        "headers",
```

Comment:
Ideally these changes would go into the other currently unmerged commits (that recently introduced these lines). Using git commit --amend and --rebase. That way the git blame/history points to the logical commit that introduced these changes.

Reply:
I think you are not commenting on the latest state, but a specific, older commit?